### PR TITLE
Upgrade eslint

### PR DESCRIPTION
**Pull request by wazolab**
Later I will apply the same rules as Clearance Frontend and same workflow (commitlint, styles linting handled by Eslint, git Hook etc ...)

Later because it will change almost all the files of the project ...

### CODE REVIEW

File: components/Fields/OpeningHours.test.js [hunk 1]
```diff
@@ -1,83 +0,0 @@
-import { createPinia } from 'pinia'
```
Comment (frodrigo) [inline components/Fields/OpeningHours.test.js:1]: This file should really be removed ?

Comment (wazolab) [reply components/Fields/OpeningHours.test.js:1]: I think it's not used anymore, we never play this test ... and it triggers lint errors (that we could fix of course).
What do you think ?

Comment (frodrigo) [reply components/Fields/OpeningHours.test.js:1]: I think it was never used automatically. But the goal is to have more tests, not less.

Comment (wazolab) [reply components/Fields/OpeningHours.test.js:1]: Alright, I agree. I've fixed the lint errors !

**Pull request merged**